packages/lesswrong/server/repos/CommentsRepo.ts

```diff
@@ -75,6 +75,54 @@ export default class CommentsRepo extends AbstractRepo<DbComment> {
     `, [limit]);
   }
 
+  async getPopularPollComments (limit: number): Promise<(DbComment)[]> {
+    return await this.manyOrNone(`
+      SELECT *
+      FROM public."Comments" AS c
+      WHERE c."parentCommentId" = 'NtsPs9wcwrpeK6KYL'
```

Comment:
I'd assign this commentId as a variable/prop that you pass in, or at least leave a comment explaining where this comes from and why.

Hardcoded _ids are generally not great for ForumMagnum as its not just us using this code. If EA Forum were to want to adopt this, it'd need to be cleaned up. So worth a few thoughts to making this reusable and forum agnostic.